t/testset_gff_bed_mapping/tr.bed

```diff
@@ -0,0 +1,6 @@
+# case1 minus strand, exon spanning
+XM_008192597.2	391	415	.	.	+
+# case2 minus strand, two exon spannings
```

Comment:
Same here... Comment says minus strand, but bed indicates plus

Reply:
this is the bed file for the transcriptome and therefore it is plus stranded.
nevertheless, the test is for the genome minus strand